server/pulp/server/db/model/dispatch.py

```diff
@@ -489,3 +490,27 @@ def __init__(self, call_request, call_report):
         self.timestamp = dateutils.now_utc_timestamp()
         self.call_request_string = str(call_request)
         self.serialized_call_report = call_report.serialize()
+
+
+class CeleryBeatLock(Document):
+    """
+    Single document collection which gives information about the current celerybeat lock.
+
+        :param celerybeat_name: string representing the celerybeat instance name
+        :type  celerybeat_name: basestring
+        :param timestamp: The timestamp(UTC) at which lock is acquired
+        :type  timestamp: datetime.datetime
```

Comment:
On 06/26/2015 04:40 PM, bmbouter wrote:
> |type| should have a single space after it.

I think this might be a personal preference thing. I usually do it the
way Shubham did.